modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/upgrade/DEServiceUpgrade.java

```diff
@@ -51,7 +51,12 @@ public void register(Registry registry) {
 				DEDataDefinitionFieldLinkUpgradeProcess());
 
 		registry.register(
-			"2.1.1", "2.2.0",
+			"2.1.1", "2.1.2",
+			new com.liferay.data.engine.internal.upgrade.v2_1_2.
+				DEDataDefinitionFieldLinkUpgradeProcess());
+
+		registry.register(
+			"2.1.2", "2.2.0",
```

Comment:
@linolaoj, why didn't you create the new upgrade step considering the last available step 2.2.0? Shouldn't we use 2.2.1 instead? Are you doing it due to backport?

Thanks in advance for your clarifications. :slightly_smiling_face:

Reply:
Hi @natocesarrego, yes, it is for backporting reasons. I chose 2.1.2 because this needs to run when the user upgrades from 7.3 to 7.4 ga1. But I can add a new step if you believe it is better.

Reply:
Thanks for your clarification, @linolaoj! :slightly_smiling_face: 

@linolaoj, if we keep the existing upgrade steps sequence and create a new upgrade step with number 2.2.1, a micro version, shouldn't it be enough?

According to the Upgrade Team docs, [either minor and micro changes can be backported](https://liferay.atlassian.net/wiki/spaces/ENGCOREUPGRADE/pages/1654719711/Upgrade+Development+Training#Schema-version), therefore I believe that we can follow the actual sequence of upgrade steps. If you can't do it for backport limitations then we should keep this pull as it is. :slightly_smiling_face: